css-showcases/src/transition-box.module.scss

```diff
@@ -1,22 +1,22 @@
-.details {
+.boxLabel {
+  font-family: monospace;
   margin: 0.125rem;
   font-size: 0.75rem;
   color: #fff;
 }
 
-.details-sm {
-  font-size: 0.55rem;
+.boxLabelSmall {
+  font-size: 0.65rem;
 }
 
-.wrapper {
+.boxesContainer {
```

Comment:
Should be dockit-row I think instead of a new class here

Reply:
this uses css modules to avoid global class names collisions, so adding `dockit-row` tag won't add value IMO.

Reply:
I know but it's just about reusing components all across the dockit instead of redefining a meaning less div with a class instead of reusing a component.